parsel/selector.py

```diff
@@ -39,8 +44,12 @@ def create_root_node(text, parser_cls, base_url=None):
     """Create root node for text using given parser class.
     """
     body = text.strip().replace('\x00', '').encode('utf8') or b'<html/>'
-    parser = parser_cls(recover=True, encoding='utf8')
-    root = etree.fromstring(body, parser=parser, base_url=base_url)
+    if parser_cls != html5parser.HTMLParser:
```

Comment:
I think it should be legible if we define the option as `==` for the case of `html5parser` and then `else` for the other options

Reply:
Hey!, thanks for review!, I also think that, i'll update it.